bats/sanity-check/20-selftests.bats

```diff
@@ -306,6 +306,8 @@ tap_negative_check() {
         fi
         test_yaml_numeric "/cpu-info/$i/logical" 'value >= 0'
         test_yaml_numeric "/cpu-info/$i/package" 'value >= 0'
+        test_yaml_numeric "/cpu-info/$i/numa_node" 'value >= -1'
+        test_yaml_numeric "/cpu-info/$i/module" 'value >= 0'
```

Comment:
the CI is right. this should be >= -1.

Reply:
No, that's a bug in the code. Linux is indeed telling us that the `cluster_id` is 65535 but that's a bad solution.